extension/content.js

```diff
@@ -1,27 +1,9 @@
-/* globals gitHubInjection, diffFileHeader */
+/* globals gitHubInjection, pageDetect, diffFileHeader */
 'use strict';
-const path = location.pathname;
-const ownerName = path.split('/')[1];
-const repoName = path.split('/')[2];
+const segments = location.pathname.split('/');
+const ownerName = segments[1];
```

Comment:
:+1: Destructuring became available in Chrome 49, so we can start using it now. (Need to bump the min Chrome version in the manifest.